pelican/tools/pelican_quickstart.py

```diff
@@ -21,8 +21,12 @@
 
 from pelican import __version__
 
-if (sys.version_info.major == 2):
-    locale.setlocale(locale.LC_ALL, '')
+locale.setlocale(locale.LC_ALL, '')
+language = locale.getlocale()[0]
```

Comment:
For consistency, can we name the variable like the others (`_DEFAULT_TIMEZONE`)? Something like `_DEFAULT_LANGUAGE`. Otherwise, looks good.